subgraphs/compound-v2/src/mappings/mappings.ts

```diff
@@ -0,0 +1,126 @@
+import { exponentToBigDecimal } from "../common/utils/utils";
+import { Address } from "@graphprotocol/graph-ts";
+import { BIGDECIMAL_ONE, DEFAULT_DECIMALS } from "../common/utils/constants";
+
+export function handleMint(event: Mint): void {
+  if (createDeposit(event, event.params.mintAmount, event.params.mintTokens, event.params.minter)) {
+    updateUsageMetrics(event, event.params.minter);
+    updateFinancials(event);
+    updateMarketMetrics(event);
+  }
+}
+
+export function handleRedeem(event: Redeem): void {
+  if (createWithdraw(event, event.params.redeemer, event.params.redeemAmount, event.params.redeemTokens)) {
+    updateUsageMetrics(event, event.params.redeemer);
+    updateFinancials(event);
+    updateMarketMetrics(event);
+  }
+}
+
+export function handleBorrow(event: Borrow): void {
+  if (createBorrow(event, event.params.borrower, event.params.borrowAmount)) {
+    updateUsageMetrics(event, event.params.borrower);
+    updateFinancials(event);
+    updateMarketMetrics(event);
+  }
+}
+
+export function handleRepayBorrow(event: RepayBorrow): void {
+  if (createRepay(event, event.params.payer, event.params.repayAmount)) {
+    updateUsageMetrics(event, event.params.payer);
+    updateFinancials(event);
+    updateMarketMetrics(event);
+  }
+}
+
+export function handleLiquidateBorrow(event: LiquidateBorrow): void {
+  if (
+    createLiquidation(
+      event,
+      event.params.cTokenCollateral,
+      event.params.liquidator,
+      event.params.seizeTokens,
+      event.params.repayAmount,
+    )
+  ) {
+    updateUsageMetrics(event, event.params.liquidator);
+    updateFinancials(event);
+    updateMarketMetrics(event);
+  }
+}
+
+export function handleMarketListed(event: MarketListed): void {
+  // create new market now that the data source is instantiated
+  CToken.create(event.params.cToken);
+  getOrCreateMarket(event, event.params.cToken);
+}
+
+export function handleNewPriceOracle(event: NewPriceOracle): void {
+  // create LendingProtocol - first function to be called in Comptroller
+  let lendingProtocol = getOrCreateLendingProtcol();
+
+  lendingProtocol._priceOracle = event.params.newPriceOracle;
+  lendingProtocol.save();
+}
+
+export function handleNewReserveFactor(event: NewReserveFactor): void {
+  let market = getOrCreateMarket(event, event.address);
+
+  // update financials in case the reserve is updated and no other compound transactions happen in that block
+  // intended for capturing accurate revenues
+  updateFinancials(event);
+
+  // get reserve factor
+  market._reserveFactor = event.params.newReserveFactorMantissa
+    .toBigDecimal()
+    .div(exponentToBigDecimal(DEFAULT_DECIMALS));
+  market.save();
+}
+
+export function handleNewCollateralFactor(event: NewCollateralFactor): void {
+  let market = getOrCreateMarket(event, event.params.cToken);
+  let newLTV = event.params.newCollateralFactorMantissa.toBigDecimal().div(exponentToBigDecimal(16));
```

Comment:
can you make `16` a named constant and document it?

Reply:
fixed

Reply:
this is fixed?